app/services/arcgis_api/geocoder.rb

```diff
@@ -35,11 +35,9 @@ class Geocoder
       ].join(','),
     }.freeze
 
-    ROOT_URL = IdentityConfig.store.arcgis_api_root_url
-    SUGGEST_ENDPOINT = "#{ROOT_URL}/servernh/rest/services/GSA/USA/GeocodeServer/suggest"
-    ADDRESS_CANDIDATES_ENDPOINT =
-      "#{ROOT_URL}/servernh/rest/services/GSA/USA/GeocodeServer/findAddressCandidates"
-    GENERATE_TOKEN_ENDPOINT = "#{ROOT_URL}/portal/sharing/rest/generateToken"
+    GENERATE_TOKEN_ENDPOINT = IdentityConfig.store.arcgis_api_generate_token_url
+    SUGGEST_ENDPOINT = IdentityConfig.store.arcgis_api_suggest_url
+    ADDRESS_CANDIDATES_ENDPOINT = IdentityConfig.store.arcgis_api_find_address_candidates_url
```

Comment:
my comment might have gotten lost, but I think we can remove these constants and just inline usage of the configs (since we removed Figaro a long time ago, reading from the config is a very cheap struct property read)